### PR TITLE
renamed torso description

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by NadiaHG** [comment]
please merge together with: https://github.com/ipa320/cob_common/pull/149

**Comment by fmessmer** [comment]
except for the commit history, the effective changes look ok...
can you pull first and then do the changes? or squash the last 4 commits into "renamed torso urdf"?

File: cob_hardware_config/cob4-1/urdf/cob4-1.urdf.xacro [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <xacro:include filename="$(find cob_description)/urdf/cob4_base/base.urdf.xacro" />
 
   <!-- torso -->
-  <xacro:include filename="$(find cob_description)/urdf/cob4_torso/torso.urdf.xacro" />
+  <xacro:include filename="$(find cob_description)/urdf/cob4_torso/torso_3DOF.urdf.xacro" />
```
Comment (floweisshardt) [inline cob_hardware_config/cob4-1/urdf/cob4-1.urdf.xacro:17]: I prefer naming it "torso_3dof"

**Comment by fmessmer** [comment]
see [my comment](https://github.com/ipa320/cob_common/pull/149#issuecomment-86938007)

**Pull request closed without merging**